tests/sqlparser_snowflake.rs

```diff
@@ -976,6 +976,27 @@ fn parse_sf_create_or_replace_with_comment_for_snowflake() {
     }
 }
 
+#[test]
+fn parse_sf_create_table_or_view_with_dollar_quoted_comment() {
+    assert!(snowflake()
+        .parse_sql_statements(
+            r#"CREATE OR REPLACE TEMPORARY VIEW foo.bar.baz (
+            "COL_1" COMMENT $$comment 1$$
+        ) COMMENT = $$view comment$$ AS (
+            SELECT 1
+        )"#
+        )
+        .is_ok());
+
+    assert!(snowflake()
+        .parse_sql_statements(
+            r#"CREATE TABLE my_table (
+            a STRING COMMENT $$comment 1$$
+        ) COMMENT = $$table comment$$"#
+        )
```

Comment:
can we use `snowflake().verified_stmt(sql)` for the tests?

Reply:
1. There is no code for storing a comment quotation. Could you clarify whether it makes sense to update all the code that handles comment?
2. Snowflake converts a dollar-quoted comment into a single-quoted comment in the DDL representation of table/view creation.

Reply:
Ah that makes sense, I think we can leave as is in that case in order to limit the scope of the PR